test/parallel.jl

```diff
@@ -21,7 +25,7 @@ for i in 1:n
     dfSom = vcat(dfSom, dfSom)
 end
 
-som2 = initGigaSOM(dfSom, 10, 10)
+@time som2 = initGigaSOM(dfSom, 10, 10)
```

Comment:
I am not sure this `@time` is necessary here.

Reply:
removed